pkgs/by-name/fi/ficsit-cli/package.nix

```diff
@@ -0,0 +1,37 @@
+{
+  lib,
+  stdenv,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+buildGoModule rec {
+  pname = "ficsit-cli";
+  version = "0.6.0";
+  commit = "5dc8bdbaf6e8d9b1bcd2895e389d9d072d454e15";
```

Comment:
Probably not needed now — most other packages simply provide the tag name (e.g. `refs/tags/v${version}` as the commit name)

Reply:
I'm confused, are you saying there's a way to get the commit from the tag?

I have it set here because the app needs it during build to fill in the version field
![2024-12-31-14:25:41](https://github.com/user-attachments/assets/9e77de72-f14a-4227-889a-4e7856811469)